src/lib/syscall-logger/src/lib.rs

```diff
@@ -88,10 +88,9 @@ pub fn log_syscall(args: TokenStream, input: TokenStream) -> TokenStream {
             ctx: &mut crate::host::context::ThreadContext,
             args: &crate::host::syscall_types::SysCallArgs,
         ) -> crate::host::syscall_types::SyscallResult {{
-            let strace_fmt_options = match ctx.process.strace_logging_options() {{
-                Some(x) => x,
+            let Some(strace_fmt_options) = ctx.process.strace_logging_options() else {{
```

Comment:
Extra pair of brackets here?

Reply:
Ah, GitHub doesn't show that it's actually part of a `format!()` string, so the first bracket is needed to escape the second one. (We should be using the syn crate here, but I didn't originally add it to avoid depending on it, but we use a lot of crates that depend on it now so I don't think that would be an issue.)